Dockerfile

```diff
@@ -17,30 +17,31 @@ ADD https://github.com/lifting-bits/remill/releases/latest/download/remill_ubunt
 # Saves a bit of space in the base image.
 # Also better for not repeating ourselves when installing remill
 RUN apt-get update && \
-    apt-get install -qqy --no-install-recommends unzip && \
-    rm -rf /var/lib/apt/lists/* && \
-    unzip remill_packages.zip && \
-    rm remill_packages.zip && \
-    mv ubuntu-${UBUNTU_VERSION}_llvm${LLVM_VERSION}_deb_package/remill-*.deb ../remill.deb && \
-    cd .. && rm -rf tmp
+    apt-get install -qqy --no-install-recommends unzip
 
 
 # Run-time dependencies go here
 FROM ${BUILD_BASE} AS base
 ARG UBUNTU_VERSION
 ARG LIBRARIES
 ARG LLVM_VERSION
+ARG CXX_COMMON_VERSION
+ARG DEBIAN_FRONTEND=noninteractive 
 RUN apt-get update && \
-    apt-get install -qqy --no-install-recommends libdbus-1-3 curl unzip python3 python3-pip python3.8 python3.8-venv python3-setuptools xz-utils && \
+    apt-get install -qqy --no-install-recommends git libdbus-1-3 curl unzip python3 python3-pip python3.8 python3.8-venv python3-setuptools xz-utils cmake && \
     rm -rf /var/lib/apt/lists/*
 
 WORKDIR /dependencies
 
+ADD https://github.com/lifting-bits/cxx-common/releases/download/v${CXX_COMMON_VERSION}/vcpkg_ubuntu-${UBUNTU_VERSION}_llvm-${LLVM_VERSION}_amd64.tar.xz vcpkg_ubuntu-${UBUNTU_VERSION}_llvm-${LLVM_VERSION}_amd64.tar.xz
```

Comment:
i believe cxx-common is already preset in this build step's base image (FROM trailofbits/cxx-common-vcpkg-builder-ubuntu:${UBUNTU_VERSION} as deps)